src/utils/intervalUpdates.ts

```diff
@@ -0,0 +1,39 @@
+import { LendBorrowDayData } from '../../generated/schema'
+import { Notional, LendBorrowTrade } from '../../generated/Notional/Notional';
+import { BigInt } from '@graphprotocol/graph-ts'
+import { getMarketIndex } from '../common';
+import { convertAssetToUnderlying } from '../accounts';
+
+export function updateLendBorrowDayData(event: LendBorrowTrade): LendBorrowDayData
+{
+    let notional = Notional.bind(event.address);
+    let timestamp = event.block.timestamp.toI32()
+    let dayId = timestamp / 86400 // rounded
+    let dayStartTimestamp = dayId * 86400
+    let currencyId = event.params.currencyId as i32;
+    let maturity = event.params.maturity;

```

Comment:
We should also store references to the currency and market in the object so we can filter on them. 

You can get the marketId and currencyId from the function you call this from.
`trade.market = currencyId.toString() + ':' + settlementDate.toString() + ':' + maturity.toString();`